cluster/gke/util.sh

```diff
@@ -284,13 +289,19 @@ function detect-node-names {
 #   ZONE
 #   CLUSTER_NAME
 # Vars set:
-#   NODE_INSTANCE_GROUP
+#   NODE_INSTANCE_GROUPS
 function detect-node-instance-group {
   echo "... in gke:detect-node-instance-group()" >&2
-  local url=$("${GCLOUD}" ${CMD_GROUP:-} container clusters describe \
+  local urls+=$("${GCLOUD}" ${CMD_GROUP:-} container clusters describe \
     --project="${PROJECT}" --zone="${ZONE}" \
     --format='value(instanceGroupUrls)' "${CLUSTER_NAME}")
-  NODE_INSTANCE_GROUP="${url##*/}"
+  urls=(${urls//;/ })
```

Comment:
No, the addition of the quotes keeps this as a single string, which breaks the "for" loop below (i.e. it returns the whole result as a single entry).

(I'm assuming the backtick was a typo).